### PR TITLE
Feature/add ingredients info to panel

### PR DESCRIPTION
**Pull request by mbenfowler**
##Description
This pr adds ingredient data to the right side panel of the recipe.

Screenshot 2023-05-18 at 5 17 01 PM

##Testing Notes
- Click on a recipe, you should see the corresponding amount and units of each ingredient required
  - If ingredients are long, they should word-wrap onto the next line
  - If ingredients list is super long, it will clip off the bottom of the page.  I don't have any scrolling implemented.
- check boxes function as expected
-  When recipe panel is closed, all ingredients are removed from the element and ready to accept the next set

### CODE REVIEW

File: src/domUpdates.js [hunk 1]
```diff
@@ -173,8 +174,20 @@ const closeRecipe = () => {
...
 };
 
+const getIngredients = currentRecipeCard => {
+  createIngredientsHTML(currentRecipeCard.ingredients);
+};
+
+const createIngredientsHTML = ingredients => {
+  ingredients.forEach(ingredient => {
+    ingredientsList.innerHTML += `
+      <label><input type="checkbox" class="ingredient-text" value="value">${ingredient.amount} ${ingredient.unit} ${ingredient.name}</label>
```
Comment (lauraguerra1) [inline src/domUpdates.js]: How would you feel about making separate label and input tags and adding `id` to input and `for` to label to connect the two? From my understanding, I think that screen readers rely on that type of connection to know which input a certain label is for. 

Let me know if that is not correct, but I think that it the standard way to go about using the two in conjunction.

File: src/styles.css [hunk 2]
```diff
@@ -256,6 +260,26 @@ h3 {
...
+  cursor: pointer;
+  transform: scale(1.1);
+}
+
+input[type=checkbox] {
+  accent-color: rgb(0, 172, 0);
+  transform: scale(2);
+  clip-path: circle(50% at 50% 50%);
+  margin-right: 20px;
+  padding: 10px;
```
Comment (lauraguerra1) [inline src/styles.css]: Defining the type for an input tag in CSS doesn't provide the same semantic meaning as defining the type within the input tag in HTML. Is this a change we could make? 

If you need any further clarification on what I am referring to let me know!

**Review by lauraguerra1 — CHANGES_REQUESTED** [review]

File: src/domUpdates.js (hunk 1 above)
Comment (mbenfowler) [reply src/domUpdates.js]: yep can def make that change - didn't that was the standard for accessibility!

File: src/styles.css (hunk 2 above)
Comment (mbenfowler) [reply src/styles.css]: Didn't I also define it in the html? Yah might need further clarification on this one!

Comment (lauraguerra1) [reply src/styles.css]: Whoopsie total oversight on my part! So sorry, you DID define it in HTML. Please disregard.

**Comment by lauraguerra1** [comment]
[[Iteration 1] Populate the Ingredients panel](https://trello.com/c/mdsEOOqS/16-iteration-1-populate-the-ingredients-panel)

**Review by taranveersingh93 — APPROVED** [review]
Works as described. Thanks

File: src/domUpdates.js [hunk 3]
```diff
@@ -218,8 +219,24 @@ const closeRecipe = () => {
...
+};
+
+const createIngredientsHTML = ingredients => {
+  ingredients.forEach((ingredient, i) => {
+    let ingredientLabelName = `ingredient${i}`
+    ingredientsList.innerHTML += `
+    <label for=${ingredientLabelName}>
+      <input id=${ingredientLabelName} type="checkbox" name="${ingredientLabelName}" />
+      ${ingredient.amount} ${ingredient.unit} ${ingredient.name}
+    </label>
```
Comment (lauraguerra1) [inline src/domUpdates.js]: Look good! I think we may just technically be missing the quotes for `for` and `id` but other than that looks good to me

**Review by lauraguerra1 — CHANGES_REQUESTED** [review]

File: src/domUpdates.js (hunk 3 above)
Comment (mbenfowler) [reply src/domUpdates.js]: done!

**Review by lauraguerra1 — APPROVED** [review]

**Pull request merged**